README.md

```diff
@@ -8,14 +8,15 @@
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/ssec/sift/master.svg)](https://results.pre-commit.ci/latest/github/ssec/sift/master)
 
 
-Satellite Information Familiarization Tool (SIFT) was designed by the Space
-Science and Engineering Center (SSEC) at the University of Wisconsin - Madison
-to support scientists during forecaster training events. It provides a
-graphical interface for visualization and basic analysis of geostationary
-satellite data.
-
-SIFT is built on open source technologies like Python, OpenGL, and PyQt5. It
-can be run from Mac, Windows, and Linux. The SIFT application is provided as
+SIFT (Satellite Information Familiarization Tool) is a visualisation tool
+for satellite data. It provides a graphical interface that can be used for
+e.g. fast visualisation, scientific data analysis, training, cal/val activities
+and operations.
+
+SIFT is built on open source technologies like Python, OpenGL, PyQt5, and
+makes use of the [Pytroll framework](https://pytroll.github.io/) for reading
+and processing the input data.
+It can be run from Mac, Windows, and Linux. The SIFT application is provided as
```

Comment:
```suggestion
SIFT (Satellite Information Familiarization Tool) is a visualization tool
for satellite data. It provides a graphical interface that can be used for
e.g. fast visualization, scientific data analysis, training, cal/val activities
and operations.

SIFT is built on open source technologies like Python, OpenGL, PyQt5, and
makes use of the [Pytroll framework](https://pytroll.github.io/) for reading
and processing the input data.
It can be run from Mac, Windows, and Linux. The SIFT application is provided as
```